react-bricks/bricks/react-bricks-ui/contacts/Map/MapClient.tsx

```
@@ -0,0 +1,50 @@
+'use client'
+
+import { Map, Marker } from 'pigeon-maps'
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)